apps/docs/src/app/platform/component-docs/platform-forms/platform-datetime-picker/platform-datetime-picker-examples/platform-datetime-picker-reactive-example.component.ts

```diff
@@ -15,6 +15,13 @@ export class PlatformDatetimePickerReactiveExampleComponent {
     requiredDateValidator: ValidatorFn[] = [Validators.required];
 
     onSubmit(): void {
+        Object.keys(this.datetimePickerForm.controls).forEach((field) => {
+            const control = this.datetimePickerForm.get(field);
+            control.markAsTouched({ onlySelf: true });
+        });
+        if (!this.datetimePickerForm.get('withAllowNull').value) {
+            this.datetimePickerForm.get('withAllowNull').setErrors({ required: true });
```

Comment:
in order to avoid ` this.datetimePickerForm.get('withAllowNull').setErrors({ required: true });` we can add `FdpFormField.required = true`

Also in case when `FdpFormField.required = true` we do not need `[validators]="requiredDateValidator"` I guess.